controller/update_pools.go

```diff
@@ -0,0 +1,54 @@
+package controller
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/blockassets/bam_agent/service"
+)
+
+// command to update the ip address for
+// pools
+// POST
+//Just POST this file:
+//{
+//"pool1":"",
+//"pool2":"",
+//"pool3":""
+//}
+// eg { "pool1":"111.2.3.4", "pool2":"112.3.4.5", "pool3":"113.4.5.6"}
+// and we update the conf.default file on the miner
+
+const defConfigPath = "/usr/app/conf.default"
+
+// Implements Controller interface
+type PutPoolsCtrl struct {
```

Comment:
The name of the file is update_pools.go, but this is PutPools... should we make the two consistent?

Reply:
Good point... Done